_episodes_rmd/08-making-packages-R.Rmd

```diff
@@ -66,27 +66,27 @@ Suggestion: organize in a logical manner so that you know which file holds which
 Let's turn our temperature conversion functions into an R package.
 
 ```{r}
-fahrenheit_to_kelvin <- function(temp_F) {
-    # Converts Fahrenheit to Kelvin
-    temp_K <- ((temp_F - 32) * (5/9)) + 273.15
-    temp_K
+fahr_to_celsius <- function(temp_F) {
```

Comment:
Is this renaming/shortening necessary? We teach to ["name and style code consistently"](https://swcarpentry.github.io/r-novice-inflammation/06-best-practices-R/index.html) and expanded the abbreviations for [these](https://github.com/swcarpentry/r-novice-inflammation/pull/297#issuecomment-361623083) [reasons](https://github.com/swcarpentry/r-novice-inflammation/pull/320).

Reply:
Because that's how it was in the image. I was changing the functions to match the image.